### ISSUE TITLE
Won't Load

### ISSUE THREAD

**Issue by mm132** [gdkchan/SPICA#64]
The program itself will not load or start when I click on the icon in the file.

**Comment by waielal** [comment]
Please don't create multiple issues on the same topic just because nobody answered. Everbody has his private life and doesn't have the time to solve every issue all the time. Please close the other issues to keep the issue tracker clean.

About your problem:
I can not reproduce it and it is most likly to be an error on your site because the last time the code was changed was on febuary 28th. 
Did you change systems recently? 
Is .NET Framework 4.6.2 installed? (Is included if you install v4.7)
I think the .NET Framework also needs DirectX. Is that installed?

This build runs on my Win7 machine just fine: https://drive.google.com/open?id=1PxSuomMgFFSs2FQQGkcxTzSjcsrZdFBM

You also could try clearing the folder "%APPDATA%\..\Local\SPICA".
If you have problems finding this folder just hit Win+R and paste it there.

**Comment by mm132** [comment]
I have a Windows 10 and I have Direct X Installed,  but I cannot find a Spica folder in appdata/local.

**Comment by waielal** [comment]
I also tried my build on an Windows 10 machine and it worked there fine too.

The Spica folder is only created, if Spica has run once successfully.

What about .Net Framework? https://www.microsoft.com/de-de/download/details.aspx?id=55170
And does the build I provided change anything in the behavior?

**Comment by mm132** [comment]
Yes,   I have .NET framework that already came with the computer as well as DirectX,   and yes the program did run successfully once.  Your build unfortunately did not do anything. 

    On Friday, 26 October 2018 10:14 AM, Waiel Al <notifications@github.com> wrote:
 

 I also tried my build on an Windows 10 machine and it worked there fine too.The Spica folder is only created, if Spica has run once successfully.What about .Net Framework? https://www.microsoft.com/de-de/download/details.aspx?id=55170
And does the build I provided change anything in the behavior?—

**Comment by waielal** [comment]
Okay, I created another build that logs every error inside a Log-file. https://drive.google.com/open?id=18RQ8pa918EIVQbnP61QcBSHfMIXZuDN6

Please try this out and provide me the log. It should be created in the same directory as the application with a name similar to Log_636762316458000555.txt

**Comment by mm132** [comment]
Tried that,   nothing happened.
    On Saturday, 27 October 2018, 9:08:23 PM NZDT, Waiel Al <notifications@github.com> wrote:  
 
 
Okay, I created another build that logs every error inside a Log-file. https://drive.google.com/open?id=18RQ8pa918EIVQbnP61QcBSHfMIXZuDN6

Please try this out and provide me the log. It should be created in the same directory as the application with a name similar to Log_636762316458000555.txt

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
   
28/10/2018 9:33:02 AM.668: Starting SPICA
28/10/2018 9:33:02 AM.752: Exception hooks set up
28/10/2018 9:33:02 AM.768: --------------------------

**Comment by waielal** [comment]
That's weird. The first line of code is "Create the log-file" before anything SPICA related is created. If even this not executed then there has to be something fundamentally wrong on you system.

What happens if you run the application as an administrator? (right click > run as administrator)

**Comment by waielal** [comment]
With the statement "fundamentally wrong" I assume that .NET Framework is installed correctly and runs correctly.

**Comment by mm132** [comment]
I have had it on this system before and it worked just fine...I think framework does run correctly,  run as administrator does nothing.
    On Sunday, 28 October 2018, 9:48:16 AM NZDT, Waiel Al <notifications@github.com> wrote:  
 
 
With the statement "fundamentally wrong" I assume that .NET Framework is installed correctly and runs correctly.

**Comment by mm132** [comment]
> With the statement "fundamentally wrong" I assume that .NET Framework is installed correctly and runs correctly.

It has worked before and framework has already been pre-installed,  and I did run as administrator but nothing happened.  Is it some other kind of problem?

**Comment by waielal** [comment]
I'm sorry, I have no idea what the problem could be. It is really weird that the program doesn't even run one line of the program...
There has to be something blocking the execution. Do you have an anti virus program? Maybe it false positive marked SPICA as a virus? Otherwise I am out of ideas...
When was the last time it ran?

**Comment by mm132** [comment]
If I recall correctly the last time it ran was in September,  it worked all-throughout September and prior to that (around September 1-21st and beyond).   I don't seem to recall the anti-virus thing being a problem.
    On Saturday, 3 November 2018, 4:19:23 PM NZDT, Waiel Al <notifications@github.com> wrote:  
 
 
I'm sorry, I have no idea what the problem could be. It is really weird that the program doesn't even run one line of the program...
There has to be something blocking the execution. Do you have an anti virus program? Maybe it false positive marked SPICA as a virus? Otherwise I am out of ideas...
When was the last time it ran?

**Comment by waielal** [comment]
> If I recall correctly the last time it ran was in September

That is not that far ago. Pretty weird...

Hmm okay.. coming back to the second build I linked, just to be on the same page.
You did these steps and the log file still was not created?

![image](https://user-images.githubusercontent.com/24419899/47948629-82c45b00-df34-11e8-9ecc-e2d94ed257d7.png)

Otherwise I'm sorry to tell you that I don't have any idea what the problem could be.

**Comment by legoj15** [comment]
Just to make sure, don't forget to unblock the ZIP before extracting, this has caused issues in the past
![capture](https://user-images.githubusercontent.com/7399802/47949794-5520e700-df1f-11e8-89df-8bffa5a9ce1a.PNG)
